lib/LaTeXML/resources/XSLT/LaTeXML-webpage-xhtml.xsl

```diff
@@ -663,7 +663,7 @@
   <xsl:template match="ltx:toclist">
     <xsl:param name="context"/>
     <xsl:text>&#x0A;</xsl:text>
-    <xsl:element name="ul" namespace="{$html_ns}">
+    <xsl:element name="ol" namespace="{$html_ns}">
       <xsl:call-template name='add_id'/>
       <xsl:call-template name='add_attributes'/>
       <xsl:apply-templates>
```

Comment:
Are toc entries *always* of the form `<a href=...>...</a>`? I noticed that in some cases latexml won't output a link (for instance `--navigationtoc=context` does not create a link towards the current page), and that would break the spec.

An easy fix is that if the entry is not a link, then it must be wrapped into a `<span>`.

Reply:
I think they do end up always at least a span

Reply:
> I think they do end up always at least a span

In that case, that's not good because

> A span element MUST be followed by an ol ordered list; it cannot be used in "leaf" li elements.

I'll do some tests this evening to check what the output is.

Reply:
> I'll do some tests this evening to check what the output is.

Ah, the generated TOC seems to be `ol>li>a>span>...` which is absolutely fine. And this kind of TOC is always of this form, as far as I know, so no problem here.